companies/caterpillar.md

```diff
@@ -0,0 +1,25 @@
+---
+fortune_rank: 62
+name: Caterpillar
+tags: ["fortune500", "construction", "machinery", "tech", "insurance", "energy"]
+image: name-in-images-folder.png
```

Comment:
```suggestion
image: catepillar.png
```

Look like this is what your file is named :) (I haven't added good instructions on this piece of the file yet, still working on that)